src/unix/core.c

```diff
@@ -1910,3 +1910,48 @@ unsigned int uv_available_parallelism(void) {
   return (unsigned) rc;
 #endif  /* __linux__ */
 }
+
+int uv__sock_reuseport(int fd) {
```

Comment:
We already have the internal `uv__set_reuse`. Perhaps it would be a good idea to rename that one to `uv__sock_reuseaddr` for clarity now.

Reply:
`uv__sock_reuseaddr` might be a bit confusing given that this function preferentially sets `SO_REUSEPORT` instead of `SO_REUSEADDR`.

Reply:
Got any other ideas?

Reply:
As confusing as `uv__sock_reuseaddr` might be, I don't think we have another choice. So I'd say we rename it `uv__sock_reuseaddr` but add some comments to explain what will really happen in that function. What do you think?